src/shared/hooks/useNSQuery.ts

```diff
@@ -44,7 +46,7 @@ const useNSQuery = ({
     getParamsFromURL(queryParamFromUrl);
 
   const url = useMemo(() => {
-    if (!query && !accessions.length) {
+    if (!(query || accessions)) {
```

Comment:
Do we want to also check that `accession` is not an empty list?

Reply:
I removed that at some point because when accessing an empty basket, it would end up returning the URL for a star query somehow. But that is not the case anymore, because I'm just not even rendering a table when the basket (or subbasket) is empty, and also probably because I've touched some other code somewhere else I guess.
I have now modified as suggested.